src/app/tests/suites/certification/Test_TC_OO_2_3.yaml

```diff
@@ -484,8 +488,8 @@ tests:
       PICS: OO.S.A4002 && OO.S.C42.Rsp
       response:
           constraints:
-              minValue: 170
-              maxValue: 230
+              minValue: 200 - PIXIT.OO.MaxCommunicationTurnaround
```

Comment:
In 11a the `OffWaitTime` countdown starts.
In first 11b `readAttribute` of `OnOff` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In second 11b `readAttribute` of `OnTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In third 11b `readAttribute` of `OffWaitTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In 12a the `OnWithTimedOff` command is called, that should not affect countdown, but it still can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In first 12b `readAttribute` of `OnOff` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In second 12b `readAttribute` of `OnTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
In third 12b `readAttribute` of `OffWaitTime` is performed that can take up to `PIXIT.OO.MaxCommunicationTurnaround`.
Additionally I think that 15% tolerance applies here, it elapsed 10 seconds since the countdown starts, 15% of 100 is 15, so in total `185`.
So in total I think it should be `185 - 7 * PIXIT.OO.MaxCommunicationTurnaround`.

Reply:
Updated